_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -0,0 +1,110 @@
+permalink: /en/newsletters/2020/08/26/
+name: 2020-08-26-newsletter
+slug: 2020-08-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a discussion about routed coinswaps and
+includes our regular sections with summaries of questions and answers
+from the Bitcoin StackExchange, releases and release candidates, and
+notable changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Discussion about routed coinswaps:** Chris Belcher [posted][belcher
+  coinswap] a design document for a routed multi-transaction coinswap
+  implementation to the Bitcoin-Dev mailing list as a follow-up to his
+  previous post in May (see [Newsletter #100][news100 coinswap]).
+  Discussion from ZmnSCPxj and Nadav Kohen focused on checking that the
+  protocol was safe both in its use of cryptography and in ensuring that
+  the expected transactions would get confirmed (rather than alternative
+  transactions attempting theft).  In general, it appeared that all
+  major concerns were addressed and that the protocol would provide
+  several advantageous privacy benefits.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.0-beta][lnd 0.11.0-beta] is now released.   This new major
+  version allows accepting [large channels][topic large channels] (by
+  default, this is off) and contains numerous improvements to its
+  backend features that may be of interest to advanced users (see the
+  [release notes][lnd 0.11.0-beta]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #14582][] and [#19743][Bitcoin Core #19743] wallet: always do partial spends FIXME:jonatack
+
+- [Bitcoin Core #19550][] adds a new `getindexinfo` RPC that lists each
+  optional index that has been enabled, whether it is fully synced to
+  the tip of the block chain, and (in case it hasn't been fully synced)
```

Comment:
I don't think parentheses quite fit here. The case of not being fully synced is equal to the fully synced case which is not in parentheses in the previous clause.

```suggestion
  the tip of the block chain, and if not fully synced,
```